doc/api/cli.md

```diff
@@ -595,6 +595,31 @@ Path to the file used to store the persistent REPL history. The default path is
 `~/.node_repl_history`, which is overridden by this variable. Setting the value
 to an empty string (`''` or `' '`) disables persistent REPL history.
 
+### `NODE_UNHANDLED_REJECTION=state`
+<!-- YAML
+added: REPLACEME
+-->
+
+Setting this environment variable allows fine grained control over the behavior
+of unhandled rejections. This may be set to either one of `SILENT`, `WARN`,
+`ERROR_ON_GC` or `ERROR` while the default behavior without using the
+environment variable is the same as `WARN`. `WARN` reflects the
+[`unhandledRejection hook`][].
+
+Setting the environment variable to `SILENT` prevents unhandled rejections
+from being logged. The [`unhandledRejection hook`][] is still triggered in case
+it is set.
+
+If set to `ERROR_ON_GC` unhandled rejections that are garbage collected are
+raised as uncaught exceptions.
+
+If set to `ERROR` any unhandled rejection is going to turn the unhandled
+rejection into an uncaught exception. [`unhandledRejection hook`][]`s` and
```

Comment:
If users want to generate core dumps when using the `ERROR` mode for `NODE_UNHANDLED_REJECTION`, would the recommended way to do that be to add a `uncaughtException` listener and if the `fromPromise` argument value is `true` then call `process.abort()`?

Reply:
I can confirm that `NODE_UNHANDLED_REJECTION=ERROR node --abort-on-uncaught-exceiption index.js` will generate a core dump.